Eskillate/Assets/Scripts/DragAndDrop/GameController.cs

```diff
@@ -18,22 +19,74 @@ void Start()
 
             _levels = new List<Level>();
 
+            var highScores = HighScoreHelper.GetHighScores(MiniGameId.DragAndDrop);
             var level1 = new Level()
             {
                 MiniGameId = MiniGameId.DragAndDrop,
                 LevelId = 1,
                 Name = "Level1",
                 Description = "This is level 1.",
-                HighScore = 0
+                HighScore = highScores[1]
             };
 
             _levels.Add(level1);
-            LoadLevel(0);
+
+            LoadHelper.LoadSceneAdditively(this, "LevelSelectionMenu", FillLevelSelectionMenu);
 
             // Add menus
             LoadHelper.LoadGenericMenus(this);
         }
 
+        private void FillLevelSelectionMenu()
+        {
+
+            var levelSelectionCanvas = GameObject.Find("LevelSelectionCanvas");
+            var scrollListContent = levelSelectionCanvas.transform.Find("ScrollList").transform.Find("ScrollListViewport").transform.Find("ScrollListContent");
+
+            foreach (Transform child in scrollListContent.transform)
+            {
+                GameObject.Destroy(child.gameObject);
+            }
+
+            int levelCountForThisLevelTrio = 0;
+            GameObject currentLevelTrio = new GameObject();
+            foreach (var level in _levels)
+            {
+                levelCountForThisLevelTrio = levelCountForThisLevelTrio % 3;
+                if (levelCountForThisLevelTrio == 0)
+                {
+                    currentLevelTrio = Instantiate(Resources.Load("Core\\Prefabs\\LevelSelection\\LevelTrio")) as GameObject;
+                    currentLevelTrio.transform.SetParent(scrollListContent.transform, false);
+                }
+                var levelGO = currentLevelTrio.transform.GetChild(levelCountForThisLevelTrio);
+                var scoreGO = levelGO.GetChild(0);
+                scoreGO.GetComponent<TMPro.TextMeshProUGUI>().text = level.HighScore.ToString();
+                var nameGO = levelGO.GetChild(1);
+                nameGO.GetComponent<TMPro.TextMeshProUGUI>().text = level.Name;
+                var starsGO = levelGO.GetChild(2);
+                var selectButtonGO = levelGO.GetChild(3);
+                selectButtonGO.GetComponent<Button>().onClick.AddListener(delegate { OnLevelSelected(level.LevelId); });
+
+                levelCountForThisLevelTrio++;
+            }
+
+            while(levelCountForThisLevelTrio != 3)
+            {
+                GameObject.Destroy(currentLevelTrio.transform.GetChild(levelCountForThisLevelTrio).gameObject);
+                levelCountForThisLevelTrio++;
+            }
+
+            // Set Y pos to 1 to be at scrolled to the top
+            scrollListContent.transform.parent.transform.parent.GetComponent<ScrollRect>().normalizedPosition = new Vector2(0, 1);
+        }
+
+        private void OnLevelSelected(int levelId)
+        {
+            UnloadLevel();
+            LoadLevel(levelId-1);
```

Comment:
Same comment as the other file. 
Btw, lots of code repetition, can you find a way to use common code. Maybe that should be done in another PR though.